front/API.ts

```diff
@@ -17,13 +17,15 @@ export type AccessToken = string;
 type FetchParams = {
 	route: string;
 	body?: Object;
-	method?: 'GET' | 'POST' | 'DELETE'
+	method?: 'GET' | 'POST' | 'DELETE',
+	// If true, No JSON parsing is done, the raw response's content is returned 
+	raw?: true;
 }
 
 const dummyIllustration = "https://i.discogs.com/syRCX8NaLwK2SMk8X6TVU_DWc8RRqE4b-tebAQ6kVH4/rs:fit/g:sm/q:90/h:600/w:600/czM6Ly9kaXNjb2dz/LWRhdGFiYXNlLWlt/YWdlcy9SLTgyNTQz/OC0xNjE3ODE0NDI2/LTU1MjUuanBlZw.jpeg";
 
 // we will need the same thing for the scorometer API url
-const baseAPIUrl = Platform.OS === 'web' ? '/api' : Constants.manifest?.extra?.apiUrl;
+const baseAPIUrl = process.env.NODE_ENV != 'development' && Platform.OS === 'web' ? '/api' : Constants.manifest?.extra?.apiUrl;
```

Comment:
I'm currently working on a true docker and dockerless dev setup for the front and it includes the support for the /api when using expo start
You can merge it but keep in mind i will change it in the near future, I will notice you